### PR TITLE
fix helper scripts

### PR DESCRIPTION
**Pull request by KalyanovD**
In case we try to install mklfpk at the same place for another system we receive message that mklfpk was already installed. To avoid that situation we may put each mklfpk in its own folder with name of system they are installed.

### CODE REVIEW

**Comment by KalyanovD** [comment]
make test

**Comment by KalyanovD** [comment]
make test

**Comment by KalyanovD** [comment]
make test

**Comment by napetrov** [comment]
In general - have no objections as far CI will be passing.  Think that this review should be done by @dr-pain

File: scripts/mklfpk.bat [hunk 1]
```diff
@@ -48,7 +48,17 @@ set FILENAME=%~3
...
+if not exist "%DST%\win\" (
+	set is_exist=true
+	if not exist "%DST%\lib\" (
+		set is_exist=true
+	) else (
+		set is_exist=false
+	)
+)
+
+if "%is_exist%" == "true" (
```
Comment (napetrov) [inline scripts/mklfpk.bat]: Will it make sense to flip true with false here?  
Variable name have opposite meaning

**Comment by KalyanovD** [comment]
make test

File: scripts/mklfpk.bat (hunk 1 above)
Comment (KalyanovD) [reply scripts/mklfpk.bat]: After conversation with @dr-pain logic was changed. This variable was excluded.

**Comment by napetrov** [comment]
for some reason public CI wasn't started. You can try to squash your commits and push as a single one

**Comment by KalyanovD** [comment]
http://intel-ci.intel.com/ea677e82-0d95-f1d6-b5f2-8cdcd4b723a1

**Pull request merged**